ci/slt/slt.sh

```diff
@@ -34,7 +34,7 @@ tests=(
     test/sqllogictest/transform/*.slt \
     test/sqllogictest/special/* \
 )
-tests_without_views=(
+tests_without_views_and_replica=(
```

Comment:
I had interpreted this as "without views" + "without replica"...
`tests_without_views_with_replica`? (optional)

Reply:
No, this is "without (views + replica)". These tests are somehow depending on the replica size, usually by reading some introspection table about it, so they have to be set to a fixed replica and can't be varied.

Reply:
Ah, okay. I guess the comment `specific replica size tested` below had confused me.